bindings/java/src/lib.rs

```diff
@@ -188,3 +234,27 @@ fn build_operator(
 
     Ok(op)
 }
+
+fn convert_map(env: &mut JNIEnv, params: &JObject) -> HashMap<String, String> {
```

Comment:
We don't need this for now. Please remove it. Let's focus on `memory` service first.

Reply:
OK, I'm remove `build_operator` as well, since this method is redundant cross bindings, we could do it properly with a sharing lib or a macro.